data/json/items/comestibles/carnivore.json

```diff
@@ -106,7 +107,7 @@
     "parasites": 0,
     "healthy": 0,
     "fun": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": [ "EATEN_HOT", "TRADER_AVOID" ]
```

Comment:
I can understand not wanting to change all the other instances but this one is in a migrated file and is the solitary occurrence and therefore confusing. It might be best to drop at least this occurrence (unless of course traders should avoid this item, why?)